sdk/tables/azure-data-tables/src/samples/java/TableServiceAsyncClientCodeSnippets.java

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.util.logging.ClientLogger;
+import java.util.HashMap;
+import java.util.Map;
+import reactor.core.publisher.Mono;
+
+/**
+ * async code snippets for the table service
+ */
+public class TableServiceAsyncClientCodeSnippets {
+    final ClientLogger logger = new ClientLogger("TableServiceAsyncClientCodeSnippets");
```

Comment:
For samples, you can use `System.out.println()`. ClientLogger is used within our own SDK. Samples and code snippets are for other people to copy and paste into their own code, so we prefer they use `System.out.`